src/views/tutorial-view/utils/index.ts

```diff
@@ -19,3 +19,14 @@ export function formatTutorialToMenuItem(
     isActive: path === currentPath,
   }
 }
+
+export function generateCanonicalUrl(
+  defaultCollectionSlug: string,
+  tutorialSlug: string
+): URL {
+  const path = getTutorialSlug(tutorialSlug, defaultCollectionSlug)
+  const base = 'https://developer.hashi-mktg.com'
```

Comment:
Perhaps we lift this up 🤔 I'm assuming this will change during the production release.

Reply:
Yeah, this seems like something that would be a good thing to put in the environment `/config` JSON files!